BuildScript.Nuke/BuildScript.Nuke.csproj

```diff
@@ -31,4 +31,17 @@
     <PackageReference Include="CycloneDX" Version="4.0.0" ExcludeAssets="all" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="bin/Debug/BuildHost-net472/**/*.*">
+      <Pack>true</Pack>
+      <PackagePath>contentFiles\any\any\BuildHost-net472;content/BuildHost-net472</PackagePath>
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </Content>
+    <Content Include="bin/Debug/BuildHost-netcore/**/*.*">
+      <Pack>true</Pack>
+      <PackagePath>contentFiles\any\any\BuildHost-netcore;content/BuildHost-netcore</PackagePath>
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </Content>
+  </ItemGroup>
+
```

Comment:
I think we are getting rid of that in anther PR so we just need to make sure it does not get reintroduced afterwards here. Maybe we should remove it here so that we can merge this PR?